pyOlog/ophyd_tools.py

```diff
@@ -196,10 +196,15 @@ def log_pos(positioners=None, extra_msg=None):
     msg += logbook_add_objects(positioners)
 
     for p in positioners:
-        try:
-            pdict['values'][p.name] = p.position
-        except DisconnectedError:
-            pdict['values'][p.name] = DISCONNECTED
+       try:
```

Comment:
I don't think this block should have been modified.

Reply:
Wow, sorry I put the new code in the wrong place.  I am not sure how I did that. I will resolve it now